src/RpcServer/RpcServer.Node.cs

```diff
@@ -18,22 +18,24 @@ partial class RpcServer
         [RpcMethod]
         protected virtual JObject GetConnectionCount(JArray _params)
         {
-            return LocalNode.Singleton.ConnectedCount;
+            LocalNode localNode = system.LocalNode.Ask<LocalNode>(new LocalNode.GetInstance()).Result;
```

Comment:
We should call this only once, not in all methods

Reply:
Fixed.